### PR TITLE
Minimize docker image

### LINKED ISSUE #43: Reduce OVA file size

### PR DESCRIPTION
**Pull request by lhw362950217**
Fixes sql-machine-learning/playground#43
Try to use alpine base image and move init code to start.sh, image size optimized from 480M to 211M

### CODE REVIEW

File: docker/mysql/Dockerfile [hunk 1]
```diff
@@ -37,6 +32,12 @@ COPY doc/datasets/popularize_churn.sql \
      doc/datasets/create_model_db.sql \
      /datasets/
 
+COPY docker/dev/find_fastest_resources.sh /usr/local/bin/
+RUN /bin/sh -c "source find_fastest_resources.sh && \
```
Comment (Yancey0623) [inline docker/mysql/Dockerfile:36]: Can we merge these two `RUN` command lines?

File: docker/mysql/start.bash [hunk 2]
```diff
@@ -59,4 +74,10 @@ done
 # file using the trick https://unix.stackexchange.com/a/185370/325629.
 mkdir -p /work && touch /work/mysql-inited
 
-sleep infinity
+
+# c.f. https://stackoverflow.com/questions/2935183/bash-infinite-sleep-infinite-blocking for BusyBox
```
Comment (Yancey0623) [inline docker/mysql/start.bash:78]: The voted answer is `sleep infinity ` from the URL, can you please add more comment to explain why remove `sleep infinity`?

File: docker/dev/find_fastest_resources.sh [hunk 3]
```diff
@@ -48,10 +49,11 @@ function find_fastest_url() {
         cur_speed=$(ping -c 4 -W 2 "$domain" | tail -1 \
                            | grep "/avg/" | awk '{print $4}'\
                            | cut -d '/' -f 2)
-        cur_speed=${cur_speed:-99999.9}
+        cur_speed=${cur_speed:-99999}
+        cur_speed=${cur_speed/.*/}
```
Comment (wangkuiyi) [inline docker/dev/find_fastest_resources.sh:53]: Need a comment here explaining this Bash syntax.

File: docker/mysql/Dockerfile [hunk 4]
```diff
@@ -21,12 +21,7 @@
 # populating datasets, and the container running sqlflow:ci can use
 # inotifywait to wait for the creation of this file before it go on
 # running tests.
-FROM ubuntu:18.04
-
-COPY docker/dev/find_fastest_resources.sh /usr/local/bin/
-
-COPY docker/mysql/install-mysql-server.bash /
-RUN /install-mysql-server.bash
+FROM alpine:3.12
```
Comment (wangkuiyi) [inline docker/mysql/Dockerfile:24]: Need a comment here to note how much space does this change save.

**Review by wangkuiyi — APPROVED** [review]
Please feel free to merge this PR and file a new one to fix the comments.

File: docker/mysql/start.bash (hunk 2 above)
Comment (lhw362950217) [reply docker/mysql/start.bash:78]: To minimize the image size, we did not install the `bash`, so we can't use `sleep infinity` here.

File: docker/dev/find_fastest_resources.sh (hunk 3 above)
Comment (lhw362950217) [reply docker/dev/find_fastest_resources.sh:53]: Ok

**Pull request merged**